### DISCUSSION TITLE
Repo Issues

### DISCUSSION THREAD

**Discussion by go4cas** [vuejs/petite-vue#40] [General]
Will the owner(s) of this repo, please enable Issues, so that the community members can log bugs/requests?

**Comment by sqllyw** [comment]
in the readme's status:

The issue list is intentionally disabled because I have higher priority things to focus on for now and don't want to be distracted. If you found a bug, you'll have to either workaround it or submit a PR to fix it yourself. That said, feel free to use the discussions tab to help each other out.

**Reply by jikkuatwork** [reply]
Thats so sad! Wish the project was maintained! I really loved it and I wanted to post some issues!
